packages/shared/lib/migration.ts

```diff
@@ -395,44 +396,87 @@ async function fetchMigratableBalance(hexAddress: string): Promise<number> {
  *
  * @method prepareMigrationLog
  *
- * @param {string} bundleHash
  * @param {string[]} trytes
  * @param {number} balance
- * @param [boolean] mine
- * @param [number] crackability
+ * @param [string] bundleHash
  *
  * @returns {void}
  */
-export const prepareMigrationLog = (bundleHash: string, trytes: string[], balance: number): void => {
-    const transactionObjects = trytes.map((tryteString) => asTransactionObject(tryteString))
-    const { bundles } = get(migration)
+export const prepareMigrationLog = (trytes: string[], balance: number, bundleHash?: string, index?: number): void => {
```

Comment:
All log entries have timestamp and bundleHash (ledger transactions) and when user retries we reset the error message and effectively "change" the history and log always has one entry eventhough user tried multiple times.  Maybe we should just add the retry migration attempt to the log. wdyt?

Reply:
Imo, I think it can be confusing to analyze later since there would be some errors for the same transaction. On the other hand, I think what you say also makes sense, since having information about all the errors is interesting. I don't know very well how we can structure the `migration log` file to consider all cases. Any ideas?